lib/js/src/protocol/_SdlProtocolBase.js

```diff
@@ -216,7 +216,7 @@ class _SdlProtocolBase {
      */
     _setVersion (version) {
         if (version > 5) {
-            this._protocolVersion = new Version('5.1.0'); // protect for future, proxy only supports v5 or lower
+            this._protocolVersion = new Version('5.4.0'); // protect for future, proxy only supports v5 or lower
```

Comment:
This does not need to be changed. This method is for setting the major version based on the packet header. The more accurate protocol version is set in the _handleStartServiceACK method based on the packet payload.